src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,98 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int MULTIPLIER = 2;
+    private int initialCapacity;
+    private int size;
+    private Node<K, V>[] array;
+
+    public MyHashMap() {
+        initialCapacity = DEFAULT_CAPACITY;
+        array = new Node[initialCapacity];
```

Comment:
```suggestion
        array = new Node[DEFAULT_CAPACITY];
```
Maybe it will be better?